packages/salesforcedx-apex-debugger/src/commands/baseCommand.ts

```diff
@@ -47,7 +51,8 @@ export abstract class BaseCommand {
         'Content-Type': 'application/json',
```

Comment:
This one is also not part of this CL but since I just saw this the other day in @GaryFNC 's PR, I will ask again here. Is there a reason why Content-Type is quoted but Accept and Authorization are not? Should we be consistent and quote everything?

Reply:
@vazexqi This is a side effect of code formatting/cleanup. Saving the file removes unnecessary quotes.